Modules/AuctionManager/GUI.lua

```diff
@@ -362,9 +376,23 @@ local function UpdateAwardValue(self)
     end
 end
 
+function AuctionManagerGUI:setInputAwardValue(v)
+    self.awardValue = tonumber(v) or 0;
+    self.top:SetStatusText("Awarding to " .. self.awardPlayer .. " for " .. self.awardValue)
+    self:Refresh()
```

Comment:
Do we need this refresh here?

Reply:
I am not sure atm. I had issues with status text not updating earlier, but refreshing the whole gui seems a bit overkill. Can test again later